SPECS-EXTENDED/libguestfs/libguestfs.spec

```diff
@@ -147,7 +147,7 @@ BuildRequires: perl(Test::Pod) >= 1.00
 BuildRequires: perl(Test::Pod::Coverage) >= 1.00
 BuildRequires: perl(Module::Build)
 BuildRequires: perl(ExtUtils::CBuilder)
-BuildRequires: perl(Locale::TextDomain)
+BuildRequires: perl-libintl-perl
```

Comment:
Might be problematic, since `texinfo`, which also provides this Perl module, is part of the worker chroot. We'll see during the build, I guess but wanted to leave this comment here, in case we need to troubleshoot.